src/main/java/frc/robot/Constants.java

```diff
@@ -15,5 +15,63 @@
 public final class Constants {
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
+
+  public static class PhotonVisionConstants {
+    public static final double kCameraHeightMeters1 = 0.0;
+    public static final double kCameraHeightMeters2 = 0.0;
+    public static final double kCameraHeightMeters3 = 0.0;
+    public static final double KCameraPitchRadians = 0.0;
+    public static final double kTargetHeightMeters1 = 0.0;
+    public static final double kTargetHeightMeters2 = 0.0;
+    public static final double kTargetHeightMeters3 = 0.0;
+    public static final double kTargetHeightMeters4 = 0.0;
+    public static final double kTargetHeightMeters5 = 0.0;
+    public static final double kTargetHeightMeters6 = 0.0;
+    public static final double kTargetHeightMeters7 = 0.0;
+    public static final double kTargetHeightMeters8 = 0.0;
+    public static double findCameraHeight(String cameraName) {
+      if (cameraName == "1") {
+        return kCameraHeightMeters1;
+      }
+      else if (cameraName == "2") {
+        return kCameraHeightMeters2;
+      }
+      else if (cameraName == "3") {
+        return kCameraHeightMeters3;
+      }
+      else {
+        throw new Error("Invalid Camera ID");
+      }
+    }
+    public static double findTargetHeight(int targetID) {
+      if (targetID == 1) {
+        return kTargetHeightMeters1;
+      }
+      else if (targetID == 2) {
+        return kTargetHeightMeters2;
+      }
+      else if (targetID == 3) {
+        return kTargetHeightMeters3;
+      }
+      else if (targetID == 4) {
+        return kTargetHeightMeters4;
+      }
+      else if (targetID == 5) {
+        return kTargetHeightMeters5;
+      }
+      else if (targetID == 6) {
+        return kTargetHeightMeters6;
+      }
+      else if (targetID == 7) {
+        return kTargetHeightMeters7;
+      }
+      else if (targetID == 8) {
+        return kTargetHeightMeters8;
+      }
+      else {
+        throw new Error("Invalid Target ID");
+      }
```

Comment:
A switch statement would be a better control structure here. A more complicated solution that would make sense in some cases (probably not here) would be to build a map data structure and use it for lookups.